crates/ruma-events/src/kinds.rs

```diff
@@ -136,6 +138,13 @@ pub struct OriginalMessageLikeEvent<C: MessageLikeEventContent> {
 
     /// Additional key-value pairs not signed by the homeserver.
     pub unsigned: MessageLikeUnsigned<C>,
+
+    #[cfg(feature = "unstable-msc4354")]
+    #[ruma_event(default)]
```

Comment:
is there a way to have the macro consider an alias for that field? (like serde rename). This might be usefull to support both developement and final id?